services/core-web/src/components/Forms/incidents/IncidentForm.js

```diff
@@ -777,7 +777,7 @@ const renderEditSaveControls = (props, isEditMode, isNewIncident) => (
   </div>
 );
 
-const IncidentForm = (props) => {
+export const IncidentForm = (props) => {
```

Comment:
Was the import in the test not working properly with the default export?

Reply:
They din't seem to work properly after some upgrades, which was odd.

Reply:
Odd.  I've been converting some elements from class based components to functional ones and have noticed these test imports get a little finicky, but I haven't seen it require this yet.  🤷